automation/docker/application.conf.ctmpl

```diff
@@ -1,17 +1,18 @@
 # To run against a local UI, set baseUrl = "http://local.broadinstitute.org/"
-{{define "USER_DOMAIN"}}{{with $env := env "ENVIRONMENT"}}{{if eq $env "dev"}}test.firecloud.org{{else if eq $env "alpha"}}test.firecloud.org{{else if eq $env "qa"}}quality.firecloud.org{{else}}firecloud.org{{end}}{{end}}{{end}}
+{{define "USER_DOMAIN"}}{{with $env := env "ENVIRONMENT"}}{{if or (eq $env "dev") (eq $env "alpha")}}test.firecloud.org{{else if eq $env "qa"}}quality.firecloud.org{{else}}firecloud.org{{end}}{{end}}{{end}}
 {{with $environment := env "ENVIRONMENT"}}
 {{with $dir := env "ROOT_DIR"}}
+{{with $local := env "LOCAL"}}
```

Comment:
I like this! 2 things:

1. Maybe it should be LOCAL_UI to be more descriptive.
2. README.md needs to be updated here: https://github.com/broadinstitute/firecloud-ui#against-a-local-ui. I just merged my changes so you should rebase or merge develop before editing.